<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>backtest_engine</class>
 <widget class="QWidget" name="backtest_engine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="loadTickersButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Tickers</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="backtestOutput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>1021</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="runBacktestButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Backtest</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
